packaging/debian/control

```diff
@@ -0,0 +1,18 @@
+Source: azure-nvme-utils
+Section: utils
+Priority: optional
+Maintainer: Chris Patterson <cpatterson@microsoft.com>
+Build-Depends: cmake, pandoc, debhelper-compat (= 12)
+Standards-Version: 4.5.0
+Homepage: https://github.com/Azure/azure-nvme-utils
+Rules-Requires-Root: no
+#Vcs-Git: https://salsa.debian.org/debian/azure-nvme.git
+#Vcs-Browser: https://salsa.debian.org/debian/azure-nvme
+
+Package: azure-nvme-utils
+Architecture: any
+Multi-Arch: foreign
+Depends: ${misc:Depends}, ${shlibs:Depends}
+Description: auto-generated package by debmake
+ This Debian binary package was auto-generated by the
+ debmake(1) command provided by the debmake package.
```

Comment:
The description should get filled out for packages that will actually be distributed, and will definitely need to be completed for submission to Debian.

For dev builds as done here, it won't matter, so this isn't a blocker for this PR IMO.

The structure of the description is described in the [debian policy](https://www.debian.org/doc/debian-policy/ch-controlfields.html#s-f-description)

Reply:
fixed, thanks!